app/controllers/FurloughPeriodQuestionController.scala

```diff
@@ -46,49 +48,51 @@ class FurloughPeriodQuestionController @Inject()(
 
   val form = formProvider()
 
-  def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
-    getRequiredAnswers(FurloughStartDatePage, FurloughStatusPage) { (furloughStart, furloughStatus) =>
-      val preparedForm = request.userAnswers.get(FurloughPeriodQuestionPage) match {
-        case None        => form
-        case Some(value) => form.fill(value)
-      }
+  def onPageLoad(): Action[AnyContent] = (identify andThen feature(FastTrackJourneyFlag) andThen getData andThen requireData).async {
```

Comment:
nothing changed here really except adding the bit `andThen feature(FastTrackJourneyFlag)`